ansible/roles/kstest-master/templates/run_tests.sh.j2

```diff
@@ -43,6 +53,17 @@ if [ ! -d ${RESULTS_RUN_PATH} ]; then
 fi
 
 
+### Create this test result dir
+
+RESULT_PATH="${RESULTS_RUN_PATH}/${START_TIME}"
```

Comment:
Isn't it better to create file with permanent name in the `RESULTS_RUN_PATH` and add start time in it? It would be than easier to specify it outside of ansible.

Reply:
Namespacing by START_TIME is there to be able to store (keep) results of multiple (serial - ie scheduled on the master) runs (identified by the START_TIME). Syncing to remote host from master is optional, and can be done any time.

Reply:
OK thanks for explanation.